src/xunit.performance.api/Profilers/Etw/EtwManagedModule.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    public sealed class EtwManagedModule
+    {
+        public long Id { get; set; }
+
+        public string ILPath { get; set; }
+
+        public string ILFileName { get; set; }
+
+        /// <summary>
+        /// TODO: Should PerformanceMonitorCounterData be exposed via a IReadOnlyDictionary?
+        /// </summary>
+        public IDictionary<int, long> PerformanceMonitorCounterData { get; set; }
```

Comment:
What are the int and long in this context?  We should probably make it clearer what these are.

Reply:
Yes, I need to clean this interfaces.

---
In reply to: [143034051](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143034051) [](ancestors = 143034051)